backend/src/server.js

```diff
@@ -0,0 +1,39 @@
+import { ApolloServer } from "apollo-server";
+import typeDefs from "./typeDefs.js";
+import { MemoryDataSource } from "./db.js";
+import { permissions } from "./permissions.js";
+import { context } from "./context.js";
+import { applyMiddleware } from "graphql-middleware";
+import { makeExecutableSchema } from "graphql-tools";
+import { mergeResolvers } from "@graphql-tools/merge";
+import { postsMutationResolver } from "./resolvers/mutations_post";
+import { usersMutationResolver } from "./resolvers/user_mutations";
+import { queryResolver } from "./resolvers/query_all";
+
+const db = new MemoryDataSource();
+
+const dataSources = () => ({ db });
+
+//load all resolvers and combine them
+const resolvers = mergeResolvers([
+  postsMutationResolver,
+  usersMutationResolver,
+  queryResolver,
+]);
+
+//combine into executable schema
+const schema = makeExecutableSchema({ typeDefs, resolvers });
+//enrichen with middleware
+const schemaWithMiddleWare = applyMiddleware(schema, permissions);
+
+//create an instance of Apollo Server
+export default class Server {
+  constructor(opts) {
+    const defaults = {
+      schema: schemaWithMiddleWare,
+      dataSources,
+      context,
+    };
+    return new ApolloServer({ ...defaults, ...opts });
+  }
+}
```

Comment:
Diese Datei wurde scheinbar aus den Übergeordneten Ordner gelöscht und hier neu erstellt, anstatt verschoben zu werden. Das ist zwar kein größeres Problem, aber führt evlt. zu Verwirrung.

Reply:
Hmm, danke, ich bin nicht sicher warum. Vielleicht, weil es sehr viele changes werden es als 2 Dateien angezeigt?
In dem Commit mit der Verschiebung wird es als verschoben angezeigt 
https://github.com/Systems-Development-and-Frameworks/code_party_kp/pull/7/commits/bf67ec8f35ebd1c3c1283a72feed65b35047d09b